object/object.go

```diff
@@ -204,6 +205,13 @@ func (e *Environment) Get(name string) (Object, bool) {
 	return obj, ok
 }
 
+// ShallowGet returns value of given key from Enviroment's map.
+func (e *Environment) ShallowGet(name string) (Object, bool) {
+	obj, ok := e.store[name]
+
+	return obj, ok
+}
```

Comment:
Added this function to be able to check if a variable was declared in the current block, without checking the outer scope.